README.md

```diff
@@ -15,6 +15,7 @@ This page contains a list of example codes written with Optuna.
 * [CatBoost](./catboost/catboost_simple.py)
 * [Chainer](./chainer/chainer_simple.py)
 * [ChainerMN](./chainer/chainermn_simple.py)
+* [CMA-ES (Warm Starting CMA-ES)](./cma/warm_starting_cma.py)
```

Comment:
I'm wondering if we should place this `warm_starting_cma.py` under `samplers/`. It's not really an "example with an ML library" given that `cmaes` is an implementation detail in Optuna.

Reply:
I agree that this example should (or might) be moved to `samplers/` directory.
But I noticed the sentence **Examples of User-Defined Sampler** in README.md. So it is a bit inconsistent because warm starting CMA-ES is not a user-defined sampler.

Reply:
Ah, you're right. It's already a confusing structure so this is not a strong opinion but what do you think of creating a new header, something along `### Examples of Samplers` and placing this item there? The actual Python script could be placed under `samplers/` I think.

Reply:
Yeah, it looks good!
I fixed it, so please check :)

Reply:
Yeah, it looks good!
I fixed it, so please check :)

Reply:
Thanks a lot!